google-cloud-bigquery/lib/google/cloud/bigquery/schema.rb

```diff
@@ -171,6 +171,24 @@ def float name, description: nil, mode: :nullable
           add_field name, :float, description: description, mode: mode
         end
 
+        ##
+        # Adds a numeric number field to the schema. Numeric is a
+        # fixed-precision numeric type with 38 decimal digits, 9 that follow the
+        # decimal point.
```

Comment:
Can we include the terms "precision" and "scale" here? For example, the [official docs](https://cloud.google.com/bigquery/docs/reference/standard-sql/data-types#numeric-type) state:

> with 38 digits of precision and 9 decimal digits of scale. Precision is the number of digits that the number contains. Scale is how many of these digits appear after the decimal point.

For most users, "precision" and "scale" belong to a familiar vocabulary that is well-defined and therefore reassuring to see.

Reply:
In the past, we have included official disclaimers on Beta / SLA exclusions. Should we be doing that here? One concern, however, is that we might not be told when to remove the statement.

> This feature is a Beta release and is not subject to any SLA. Please see Launch stages for further information about a Beta release.